src/test/jtx/impl/JSONRPCClient.cpp

```diff
@@ -140,10 +140,57 @@ class JSONRPCClient : public AbstractClient
         Json::Reader jr;
         Json::Value jv;
         jr.parse(buffer_string(res.body.data()), jv);
-        if(jv["result"].isMember("error"))
-            jv["error"] = jv["result"]["error"];
-        if(jv["result"].isMember("status"))
-            jv["status"] = jv["result"]["status"];
+        if(jv[jss::result].isMember(jss::error))
+            jv[jss::error] = jv[jss::result][jss::error];
+        if(jv[jss::result].isMember(jss::status))
+            jv[jss::status] = jv[jss::result][jss::status];
+        return jv;
+    }
+
+    Json::Value
+    invoke(Json::Value const& cmd) override
```

Comment:
The other `invoke` does some extra work if [`rpc_version_ == 2` ](https://github.com/ripple/rippled/pull/1978/commits/da75db783e7ce3c7eddf5f5119d4597aff82b6df#diff-50358e91d80a41d6c85b73d5921a61f8R121).  Does this version need to do the same, or is the expectation that the caller will handle that?